Collections/Collections.pier

```diff
@@ -91,36 +91,36 @@ optimized or occasionally even forbidden by subclasses of ==Collection==.
-;==Keyed:==
+;""Indexable:""
+:Most sequenceable collections are also indexable, that is, elements can be retrieved with ==at:==. ==Array== is the familiar indexable data structure with a fixed size; ==anArray at: n== retrieves the n^^th^^ element of ==anArray==, and ==anArray at: n put: v== changes the n^^th^^ element to ==v==. ==LinkedList==s and ==SkipList==s are sequenceable but not indexable, that is, they understand ==first== and ==last==, but not ==at:==.
+;""Keyed:""
 :Instances of ==Dictionary== and its subclasses are accessed by keys instead of indices.
-;==Mutable:==
+;""Mutable:""
 :Most collections are mutable, but ==Interval==s and ==Symbol==s are not. An ==Interval== is an immutable collection representing a range of ==Integer==s. For example, ==5 to: 16 by: 2== is an interval that contains the elements 5, 7, 9, 11, 13 and 15. It is indexable with ==at:==, but cannot be changed with ==at:put:==.
-;==Growable:==
+;""Growable:""
 :Instances of ==Interval== and ==Array== are always of a fixed size. Other kinds of collections (sorted collections, ordered collections, and linked lists) can grow after creation. The class ==OrderedCollection== is more general than ==Array==; the size of an ==OrderedCollection== grows on demand, and it has methods for ==addFirst:== and ==addLast:== as well as ==at:== and ==at:put:==.
-;==Accepts duplicates:==
+;""Accepts duplicates:""
 :A ==Set== will filter out duplicates, but a ==Bag== will not. ==Dictionary==, ==Set== and ==Bag== use the ==\=== method provided by the elements; the ==Identity== variants of these classes use the ==\=\=== method, which tests whether the arguments are the same object, and the ==Pluggable== variants use an arbitrary equivalence relation supplied by the creator of the collection.
-;==Heterogeneous:==
+;""Heterogeneous:""
 :Most collections will hold any kind of element. A ==String==, ==CharacterArray== or ==Symbol==, however, only holds ==Character==s. An ==Array== will hold any mix of objects, but a ==ByteArray== only holds \lct{Byte}s, an ==IntegerArray== only holds ==Integer==s and a ==FloatArray== only holds ==Float==s. A ==LinkedList== is constrained to hold elements that conform to the ==Link== > ==accessing== protocol.
 
 !!Implementations of collections
 
 @sec:implementation
 
-*Some collection classes categorized by implementation technique.>file://figures/CollectionsByImpl|label=collsByImpl*
++Some collection classes categorized by implementation technique.>file://figures/CollectionsByImpl.png|width=100|label=collsByImpl+
 
 These categorizations by functionality are not our only concern; we must also
-consider how the collection classes are implemented. As shown in Figure
-*fig:collsByImpl*, five main implementation techniques are employed.
+consider how the collection classes are implemented. As shown in Figure *fig:collsByImpl* , five main implementation techniques are employed.
```

Comment:
Hmm, this reference doesn't seem to be working. I wonder why? In the figure link above, the `|label=collsByImpl+` seems correct. But when compiled, this reference shows up as 'As shown in Figure ??'.